ntoskrnl/ps/query.c

```diff
@@ -1902,7 +1902,7 @@ NtSetInformationProcess(IN HANDLE ProcessHandle,
             /* Only supported on x86 */
 #if defined (_X86_)
             Ke386SetIOPL();
-#else
+#elif !defined(_M_AMD64)
```

Comment:
Is this because
* we have stuff (incorrectly) relying on this SetInfo call to succeed,
* AMD64 should support this call, but it's unimplemented in ROS and we want to fake success, or
* AMD64 supports this call but does nothing (i.e. this is the correct implementation)?

Reply:
On 2k3 it returns STATUS_SUCCESS, on Vista and later it returns STATUS_NOT_IMPLEMENTED. I'll add a comment.

Reply:
I added an appcompat version check (doesn't work yet, but will be once the appcompat stuff is merged) to behave appropriately based on the appcompat version.